src/List.js

```diff
@@ -13,7 +13,28 @@ import { Link } from 'react-router-dom';
 import { NavigationMenu } from './NavigationMenu';
 import { useHistory } from 'react-router-dom';
 
-const convertToDays = (num) => num / 1000 / 60 / 60 / 24;
+const convertToDays = (num) => Math.round(num / 1000 / 60 / 60 / 24);
+
+const daysSinceLastPurchaseOrCreationTime = (item) =>
+  convertToDays(new Date() - (item.lastPurchasedDate || item.creationTime));
+
+const itemIsInactive = (item) =>
+  daysSinceLastPurchaseOrCreationTime(item) > 2 * item.previousEstimate ||
+  item.totalPurchases === 1;
+
+const itemState = (item) => {
+  const daysToBuy =
```

Comment:
I just tried to add four items, one of which was "soon" and two were "kind of soon" and one was "not soon". However, all ended up as the "kind of soon" color.

I believe that this is because the second `else if` statement encompasses all of our initial options. 
We can fix this by changing it to 

```javascript
if (itemIsInactive(item)) {
    return 'inactive';
  } else if (daysToBuy <= 7) {
    return 'soon';
  } else if (daysToBuy > 7 && daysToBuy < 30) {
    return 'kind-of-soon';
  }
  return 'not-soon';
```

Reply:
i think there is an issue with the acceptance criteria as described in the issue, but agree with this.  also, could maybe remove the "else"s